docs/tools/apis/indexer-api.md

```diff
@@ -5,60 +5,1073 @@ sidebar_position: 2
 
 # Indexer API
 
-:::info 🔓 Private Beta access
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import MainnetIcon from '@site/static/img/icons/lukso-signet-fuschia.svg';
+import TestnetIcon from '@site/static/img/icons/lukso-signet-yellow.svg';
 
-This feature is currently in private beta. If you are interested, please [fill in this form](https://forms.gle/rhWA25m3jjuPNPva9).
+The LUKSO Indexer provides a powerful way to query Universal Profiles, assets, LSP7 tokens, LSP8 NFTs, and metadata from the LUKSO blockchain. It offers flexible querying capabilities with support for filtering and real-time subscriptions.
```

Comment:
How about adding a screenshot of the Envio playground before or after this paragraph?
Just to make the page more attractive? Couldn't we even embed the Envio playgroun in an iframe? 🤩

<img width="958" height="406" alt="image" src="https://github.com/user-attachments/assets/6a6f6468-3c6c-4858-9c9a-daf0d3aa767c" />